backend/lib/components/Geo/gmaps/index.js

```diff
@@ -50,8 +54,49 @@ const getLocationByLatLng = async (lat, lng) => {
   };
 };
 
+const getHealthFacilityPlaces = async (lat, lng) => {
+  const MAX_HOSPITAL_FACILITIES = 5;
+  const MAX_DOCTOR_FACILITIES = 5;
+  const location = [lat, lng];
+  
+  var places = [];
```

Comment:
Same as above.

Reply:
I think that the constants here are special purpose for the health facilities only, as is the places array.   Would like to leave these here.

Reply:
Would also like to leave the **require** constants outside, as they could potentially be reused for the module in other functions/endpoints.

Reply:
yeah definitely leave that const to require outside, but let's definitely move the `places` array inside the method, also change any `var` to `let`. If we needed the variable `places` then we could use the method to get that.

Reply:
Ok - done